reference/commands/consumer/config.rst

```diff
@@ -108,7 +132,26 @@ need to specify them.
 
       $ conan config install http://url/to/some/config.zip
 
-  Conan config command stores the specified URL in the conan.conf ``general.config_install`` variable.
+
+- Install the configuration from a URL, but only getting the files inside a *origin* folder
+  inside the zip file, and putting them inside a *target* folder in the local cache:
+
+  .. code-block:: bash
+
+      $ conan config install http://url/to/some/config.zip -sf=origin -tf=target
+
+- Install configuration from 2 different zip files from 2 different urls, using different source
+  and target folders for each one, then update all:
+
+  .. code-block:: bash
+
+      $ conan config install http://url/to/some/config.zip -sf=origin -tf=target
+      $ conan config install http://url/to/some/config.zip -sf=origin2 -tf=target2
+      $ conan config install http://other/url/to/other.zip -sf=hooks -tf=hooks
+      # Later on, execute again the previous configurations cached:
+      $ conan config install
+
+  It's not needed to specify any argument, it will iterate previously stored configurations in *config_install.json*, executing them again.
```

Comment:
Would it be interesting to document the file *config_install.json* in reference/configuration_files so anyone can create one and distribute the "bundle" Conan config?

Reply:
We agreed that for this release, we wanted *config_install.json* to be a cache only, and not be shared for "bundle" conan config. If we want this, we should discuss it, add it in a next release (because support is necessary, as if you get a new *config_install.json* you might need to recurse and call it again).